resources/qml/ActionButton.qml

```diff
@@ -110,11 +112,9 @@ Button
         z: backgroundRect.z - 1
     }
 
-    ToolTip
+    Cura.ToolTip
     {
         id: tooltip
-        text: ""
-        delay: 500
-        visible: text != "" && button.hovered
+        show: button.hovered
```

Comment:
The current wording of "show" feels more like it's a function than a variable. I think this needs to be refactored a bit. 

I actually like the previous version better ( `visible: text != "" && button.hovered`)